WordPress/wordcamp.org
# Enhance next gen event budget tool

See #976

This PR replaced, removed, and updated certain fields as requested in the ticket.
The modifications for `Days` and `Hours` will be in a separate PR, as they will involve more extensive changes to the logic than expected and this PR already has enough changes.



### Screenshots

| Central Network | Event Network |
| -----------------|----------------|
| ![image](https://github.com/WordPress/wordcamp.org/assets/18050944/9814d522-5c21-4069-aa3a-16824fbe1740) | <img width="1120" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/18050944/f06369d4-c4f0-45e5-8b7b-d3b52a011360"> ![image](https://github.com/WordPress/wordcamp.org/assets/18050944/7851c680-4447-4c9f-a309-92e37b07072d) |
| ![image](https://github.com/WordPress/wordcamp.org/assets/18050944/85317325-4313-4f84-9641-a0d49940eb77) | ![image](https://github.com/WordPress/wordcamp.org/assets/18050944/92943dcb-4e8e-404f-a090-8b39c47b87f9) |
| ![image](https://github.com/WordPress/wordcamp.org/assets/18050944/b481f417-114b-4f13-9efe-d5eff61e246e) | ![image](https://github.com/WordPress/wordcamp.org/assets/18050944/3ab85225-f99d-4fc5-be91-1b9f754086cc) |

  

### How to test the changes in this Pull Request:

1. Create a new (a) og WordCamp site (b) next-gen event site through `WordCamps` >> `Add new`
2. In the dashboard of both sites, go to Budgets.
3. Check if the content is the same as the screenshots above demonstrate as well as requested in the ticket.
